Changelog.md

```diff
@@ -1,3 +1,10 @@
+### Development
+
+Bug Fixes:
+
+* Correctly run `before(:suite)` (and friends) in the context of an example
```

Comment:
I think the example group vs instance of the example group thing still matters, though.